lib/engine/game/g_1822_mx/game.rb

```diff
@@ -590,8 +647,15 @@ def close_p16
           company.close!
         end
 
-        def company_status_str
-          ''
+        def company_status_str(company)
+          bidbox_minors.each_with_index do |c, index|
```

Comment:
(bidbox_minors.with_index + bidbox_connessions.with_index).flatten.each ...

Reply:
Changed to something cleaner